src/UIView/UIView+Constraints.m

```diff
@@ -105,22 +105,50 @@ - (void)constrainView:(UIView *)view toHeight:(CGFloat)height {
     [self addConstraint:heightConstraint];
 }
 
+- (void)constrainView:(UIView *)view toAspectRatio:(CGFloat)aspectRatio {
+    NSLayoutConstraint *aspectRationConstraint = [NSLayoutConstraint constraintWithItem:view
```

Comment:
I think we could pass this through the `constrainView: attribute: toView` function.  That way, we minimize the functions that actually set constraints and its easier to make sure things like `translatesAutoresizing...` are all set.